torch/onnx/utils.py

```diff
@@ -1663,6 +1667,15 @@ def _export(
     return torch_out
 
 
+@_beartype.beartype
+def _add_onnxscript_fn(model_proto: bytes) -> bytes:
+    model = onnx.load_model_from_string(model_proto)
```

Comment:
It doesn't. I will have to figure out how to get >2GB models working.

Reply:
@BowenBao I did some dig out in >2GB scenario. I have some questions.

1. I found that export.cpp actually handles >2GB situation. It seems to save tensor proto in the same directory with onnx file, but in separate external data files, which is an ONNX suggesting way based on ONNX repo. And I figure in this circumstance, the return proto should be <2GB by then, as we already separately save the tensors from model. Otherwise, it's no benefit of doing that.
2. I think ONNX repo doesn't test 2GB case, and neither do we. I think it's because of CI? Do I need to add test case for 2GB after I implement this?

Reply:
I guess this is hard to verify anyway. I ended up adding a block for >2GB model to load without tensor files just in case my assumption that **model bytes is impossible > 2GB in this point** is inaccurate. LMK what you think.

Reply:
Tested with make up model > 2GB with @justinchuby , and the behavior is super weird in `torch.onnx.export` lol

Reply:
The detail is in description.